utils/python/CIME/SystemTests/system_tests_common.py

```diff
@@ -82,16 +93,43 @@ def coupler_log_indicates_run_complete(self):
         return False
 
     def report(self):
-        newestcpllogfile = self._getlatestcpllog()
+        newestcpllogfile = self._get_latest_cpl_log()
         self._check_for_memleak(newestcpllogfile)
 
+    def _component_compare_move(self, suffix):
```

Comment:
encapsulate component_compare_move.sh calls and component_compare_test.sh calls in these two methods. We can switch these to pure python later.